### PR TITLE
chore(docs): use icons on landing page

### PR DESCRIPTION
**Pull request by tmorehouse**
### Describe the PR

Use new icons components on landing page where possible

### PR checklist

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Enhancement
- [ ] ARIA accessibility
- [x] Documentation update
- [ ] Other (please describe)

**Does this PR introduce a breaking change?** (check one)

- [x] No
- [ ] Yes (please describe)

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch, **not** the `master` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (i.e. `[...] (fixes #xxx[,#xxx])`, where "xxx" is the issue number)
- [x] It should address only one issue or feature. If adding multiple features or fixing a bug and adding a new feature, break them into separate PRs if at all possible.
- [x] The title should follow the [**Conventional Commits**](https://www.conventionalcommits.org/) naming convention (i.e. `fix(alert): not alerting during SSR render`, `docs(badge): update pill examples`, `chore(docs): fix typo in README`, etc). **This is very important, as the `CHANGELOG` is generated from these messages.**

**If new features/enhancement/fixes are added or changed:**

- [ ] Includes documentation updates (including updating the component's `package.json` for slot and event changes)
- [ ] Includes any needed TypeScript declaration file updates
- [ ] New/updated tests are included and passing (if required)
- [x] Existing test suites are passing
- [x] CodeCov for patch has met target
- [x] The changes have not impacted the functionality of other components or directives
- [ ] ARIA Accessibility has been taken into consideration (Does it affect screen reader users or keyboard only users? Clickable items should be in the tab index, etc.)

**If adding a new feature, or changing the functionality of an existing feature, the PR's
description above includes:**

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

### CODE REVIEW

File: docs/pages/index.vue [hunk 1]
```diff
@@ -305,34 +252,9 @@
...
-                <line x1="12" y1="21" x2="12" y2="12" />
-                <line x1="12" y1="8" x2="12" y2="3" />
-                <line x1="20" y1="21" x2="20" y2="16" />
-                <line x1="20" y1="12" x2="20" y2="3" />
-                <line x1="1" y1="14" x2="7" y2="14" />
-                <line x1="9" y1="8" x2="15" y2="8" />
-                <line x1="17" y1="16" x2="23" y2="16" />
-              </svg>
-
+              <b-icon icon="tools" width="2.5em" height="2.5em" class="mx-auto d-block mb-3"></b-icon>
```
Comment (jackmu95) [inline docs/pages/index.vue]: I would suggest to use the `gear` or `gear-fill` icon.

Comment (jackmu95) [reply docs/pages/index.vue]: @tmorehouse Other than that I'am fine with the changes.

Comment (tmorehouse) [reply docs/pages/index.vue]: OK.. trying the `gear-fill` icons

Comment (jackmu95) [reply docs/pages/index.vue]: Maybe `gear` fits better with the other outline icons.

Comment (tmorehouse) [reply docs/pages/index.vue]: Yeah

Comment (jackmu95) [reply docs/pages/index.vue]: Much better.

**Pull request merged**